flowcraft/generator/utils.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+try:
+    import generator.error_handling as eh
+    from generator.process_details import colored_print
+except ImportError:
+    import flowcraft.generator.error_handling as eh
+    from flowcraft.generator.process_details import colored_print
+
+
+def get_nextflow_filepath(log_file, error_mode):
```

Comment:
I don't understand why the `error_mode` needs to be an argument of this function. It could just receive the log file and be simpler. Then, inside the function an error could be raised. It doesn't need to be InspectError or ReportError, since this is a type of error that is common to both.

Reply:
We wanted to throw a specific error depending on the mode since they are available instead of a more general one. As you said, the error is the same, but the it is triggered by different modes.

Reply:
Accomodating @odiogosilva suggestions is only a matter of adding a new type error but we wanted to keep the error relevant to the mode of execution. In hindsight it type of error has no effect on the error message the user sees.